src/components/layout/Footer.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <footer class="mavedb-dataset-footer">
+    <div class="mavedb-footer-content">
+      <div class="mavedb-footer-logo">
+        <router-link to="/" class="mavedb-logo">
+          <img src="@/assets/logo-mavedb-transparent.png" alt="MaveDB" />
+        </router-link>
+      </div>
+      <div class="mavedb-footer-links">
+        <template v-for="(link, idx) in linksToShow" :key="idx">
+          <router-link v-if="link.type === 'router'" :to="link.to">{{ link.label }}</router-link>
+          <a v-else :href="link.href" :target="link.target || '_self'" :rel="link.rel || undefined">{{ link.label }}</a>
+        </template>
+      </div>
+    </div>
+  </footer>
+</template>
+
+<script lang="ts">
+import useAuth from '@/composition/auth'
+import { useRoute } from 'vue-router'
+
+interface FooterLink {
+  type: 'router' | 'external';
+  label: string;
+};
+
+export interface RouterLink extends FooterLink {
+  type: 'router';
+  to: string;
+}
+
+export interface ExternalLink extends FooterLink {
+  type: 'external';
+  href: string;
+  target?: string;
+  rel?: string;
+}
+
+export default {
+  name: 'DataSetFooter',
+  computed: {
+    linksToShow() {
+      return this.footerLinks
+    }
+  },
+  setup: () => {
+    const {userIsAuthenticated} = useAuth()
+    const route = useRoute()
+
+    // Route-based extra links
+    let routeFooterLinks: (RouterLink | ExternalLink)[] = []
+    if (route.name === 'scoreSet' || route.name === 'experiment' || route.name === 'experimentSet') {
+      routeFooterLinks.push({
+        type: 'external',
+        href: `https://mavedb.zulipchat.com/#narrow/channel/511832-dataset-issues/topic/${route.params.urn}`,
+        label: 'Report a Dataset issue',
+        target: '_blank',
+        rel: 'noopener'
+      })
+    }
+    // Add more route checks as needed
+
+    // Defined separately for typing support
+    const authFooterLink: RouterLink[] = userIsAuthenticated.value
+      ? [{ type: 'router', to: '/settings', label: 'Account' }]
+      : []
+
+    /*
+      Default links that are always shown in the footer.
+      These can be added to by passing the `additionalFooterLinks` prop.
+    */
+    const footerLinks: (RouterLink | ExternalLink)[] = [
+        ...routeFooterLinks,
+        { type: 'external', href: 'https://zulip.mavedb.com', label: 'Chat', target: '_blank', rel: 'noopener' },
+        { type: 'router', to: '/documentation', label: 'Docs' },
```

Comment:
Does it link to the Documentation page? I think the path is `/docs` except there'll be another documentation page. This one shows me "No match found for location with path '/documentation'" in Console.

Reply:
Thanks, fixed